stellargraph/connector/neo4j/graph.py

```diff
@@ -0,0 +1,97 @@
+__all__ = ["Neo4jStellarGraph"]
+
+import numpy as np
+import scipy.sparse as sps
+from ...core.experimental import experimental
+
+
+@experimental(reason="the class is not fully tested and lacks documentation")
```

Comment:
it's probably worth creating an issue (and referencing it here) to track the remaining work.